src/meta/src/controller/catalog.rs

```diff
@@ -813,7 +814,11 @@ impl CatalogController {
     }
 
     /// `finish_streaming_job` marks job related objects as `Created` and notify frontend.
-    pub async fn finish_streaming_job(&self, job_id: ObjectId) -> MetaResult<NotificationVersion> {
+    pub async fn finish_streaming_job(
```

Comment:
Not related to this PR, please help to move this function to `streaming_job.rs`? 🙏